frontend/src/helpers/row.helper.ts

```diff
@@ -0,0 +1,15 @@
+
+export interface DataRowModel {
```

Comment:
Czy to nie powinno być w osobnym pliku?

Reply:
Tak, powinno. Zapomniałem to wynieść. Dzięki , że zwróciłeś uwagę :)